Find a functionally equivalent code:android.media.AudioService.playSoundEffectVolume:COMMENT
Method Modifier: public      
Comment:/**
 * @see AudioManager#playSoundEffect(int, float)
 */

@@ -1,3 +1,7 @@
 {
+    if (effectType >= AudioManager.NUM_SOUND_EFFECTS || effectType < 0) {
+        Log.w(TAG, "AudioService effectType value " + effectType + " out of range");
+        return;
+    }
     sendMsg(mAudioHandler, MSG_PLAY_SOUND_EFFECT, SENDMSG_QUEUE, effectType, (int) (volume * 1000), null, 0);
 }

